spec/support/split_checkout_helper.rb

```diff
@@ -44,4 +44,13 @@ def fill_out_shipping_address
       select "New South Wales", from: "State"
     end
   end
+
+  def fill_out(notes)
```

Comment:
The name `fill_out` is very generic and doesn't tell us what it fills out. You could think that it fills out anything but it's restricted to one field.  What do you think about `fill_notes(text)`?

Reply:
Sounds great, easier to read :+1: